Stellar-ledger-entries-limits-v2.x

```diff
@@ -12,21 +13,34 @@ enum StatsOpType
     PAYOUT = 5
 };
 
+// `LimitsV2Entry` represents all the details of limits attached to the particular account
```

Comment:
Account? Nope. Account is optional, same as account role. Maybe, hierarchy needs to be briefly described?